api/model.go

```diff
@@ -1640,7 +1713,7 @@ func toVolumeRecurringJobCollection(recurringJobs map[string]*longhorn.VolumeRec
 	return &client.GenericCollection{Data: data, Collection: client.Collection{ResourceType: "volumeRecurringJob"}}
 }
 
-func toBackupTargetResource(bt *longhorn.BackupTarget) *BackupTarget {
+func toBackupTargetResource(bt *longhorn.BackupTarget, apiContext *api.ApiContext) *BackupTarget {
```

Comment:
Why is the apiContext argument added?  It doesn't seem to be used in this function.

Reply:
You are right. It is not necessary.
Removed. Thanks